deeplake/core/tensor.py

```diff
@@ -714,6 +718,23 @@ def __iter__(self):
                 i, is_iteration=not isinstance(self.index.values[0], list)
             )
 
+    @property
+    def is_empty_tensor(self):
+        if self.meta.is_link:
```

Comment:
not sure why we need separate logic for links?

Reply:
because meta is not updated for link tensors in case `create_shape_tensor=False` was specified. So if meta shape is empty, it could also be this case